src/libsyntax_ext/format.rs

```diff
@@ -68,7 +68,6 @@ struct Context<'a, 'b:'a> {
     name_positions: HashMap<String, usize>,
 
     /// Updated as arguments are consumed or methods are entered
```

Comment:
Oops, sure. Note that `next_arg` is still used for verifying types for implicit positional args, so I'll only delete the reference to methods.